### ISSUE TITLE
There is no /models/model.tflite file available

### ISSUE THREAD

**Issue by abdul-rehman-2050** [larswaechter/quickdraw-cnn#3]
Can you please link to the tflite model?

**Comment by larswaechter** [comment]
You can find it [here](https://github.com/larswaechter/quickdraw-cnn/tree/main/webapp/static/models) :)

**Comment by abdul-rehman-2050** [comment]
Ooh my bad. Thank you so much for this. This is a great work. I really appreciate this. Do we have keras model available as well? How much accuracy difference do we after converting to the tensorflow lite?

**Comment by larswaechter** [comment]
Unfortunately I don't have the Keras model any more, so you have to train it on your own. But the code is in the according Jupyter Notebook included. The Keras model has an accuracy of ~61%. I didn't calculate the accuracy after converting to TF Lite.

**Comment by hyeonhh** [comment]
Hi :) Can I use model.tflite file with android App?? @larswaechter
